xbmc/guilib/guiinfo/SystemGUIInfo.cpp

```diff
@@ -321,6 +321,48 @@ bool CSystemGUIInfo::GetLabel(std::string& value, const CFileItem *item, int con
       return true;
 #endif
 
+#if defined(TARGET_ANDROID)
```

Comment:
HDR is supported on Linux and Windows too. There is not reason to show these infos only on Android. 

On platforms that not support any HDR type "none" informative string is fine too.

`IsHDRDisplay()` can be called on all platforms.

Reply:
Yea I'm aware, but there's no implementations for the new methods.
So for these platforms it would be assumed HDR10, as far as I know.

Reply:
Don't worry: as `DisplaySupportsDolbyVision()` is defined on WinSystem.h and returns false by default, on all platforms where is not supported or is not implemented yet, `DisplaySupportsDolbyVision` can be called anyway and returns false.

Reply:
Yes, if `IsHDRDisplay()` is false then the System Info already displays `None` for the HDR support.